extensions/amp-soundcloud/amp-soundcloud.md

```diff
@@ -0,0 +1,46 @@
+<!---
+Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+### <a name="amp-soundcloud"></a>amp-soundcloud
+
+Displays a Soundcloud clip
+
+Example:
+```html
+<amp-soundcloud height=657
+    layout="fixed-height"
+    data-trackid="243169232"
+    data-visual="true"
+    data-color="ff5500"></amp-soundcloud>
```

Comment:
from the code data-visual + data-color isnt supported right? (we have an else-if on the appending of data-color)